core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/metadata/OperatorMetadataGenerator.scala

```diff
@@ -48,7 +49,7 @@ object OperatorMetadataGenerator {
   def main(args: Array[String]): Unit = {
     // run this if you want to check the json schema generated for an operator descriptor
     // replace the argument with the class of your operator descriptor
-    println(generateOperatorJsonSchema(classOf[LocalCsvFileScanOpDesc]).toPrettyString)
+    println(generateOperatorJsonSchema(classOf[LinearRegressionOpDesc]).toPrettyString)
```

Comment:
just FYI, the new code doesn't need to generate a `.json` file anymore, it's generated on the fly instead, I made this simply to check whether the json schema looks right